.gitignore

```diff
@@ -13,6 +13,7 @@
 
 # Dependency directories (remove the comment below to include it)
 # vendor/
+
```

Comment:
Either needs to be a part of fa4846722d5012aa47fde3c05c6e639bc7f118a0, or not be changed at all.